src/src/fileLoger/hillel/FileLogger.java

```diff
@@ -0,0 +1,53 @@
+package fileLoger.hillel;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class FileLogger {
+    private FileLoggerConfiguration config;
+
+    public FileLogger(FileLoggerConfiguration config) {
+        this.config = config;
+    }
+
+    public void debug(String message)  {
+        if (config.getLevel().compareTo(LoggingLevel.DEBUG) >= 0) {
```

Comment:
This `config.getLevel().compareTo(LoggingLevel.DEBUG) >= 0` can be extracted to a separate method, name it `isDebugEnabled()`